frontend/apps/web/components/ListBox/ListBox.tsx

```diff
@@ -90,10 +93,7 @@ export default function ListBox<TData>(props: Props<TData>): ReactElement {
                   rowVirtualizer.measureElement(node);
                 }} // measure dynamic row height
                 key={row.id}
-                style={{
-                  transform: `translateY(${virtualRow.start}px)`,
-                }}
-                className="items-center flex absolute w-full px-2"
```

Comment:
the absolute positioning I'm pretty sure is here so that the virtual scrolling works properly.